src/App.tsx

```diff
@@ -1,34 +1,80 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './app.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { getFilteretTodos } from './helpers';
+import { TodoStatus } from './types/TodoStatus';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [todoStatus, setTodoStatus] = useState(`${TodoStatus.all}`);
```

Comment:
The initial state for `todoStatus` is set to a string template literal. You can just use `TodoStatus.all` without the template literal.

Reply:
if i set TodoStatus.all i have some trouble, because i need to use setTodoStatus status with event.target.value and its string. notTodoStatus
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/130560419/7efab01f-45b2-490f-b372-446161f90dde)